docs/data/routes/docs/client-frameworks/nextjs/nextjs-static-export/en.md

```diff
@@ -0,0 +1,42 @@
+---
+name: nextjs-static-export
+routeTemplate: ./data/component-templates/article.yml
+title: Next.js static export
+---
+
+# Next.js Static HTML Export
+
+Using JSS Next.js sample app you can run `jss export` that allows you to export your app to static HTML, which can be run standalone without the need of a Node.js server. Read more about Next.js [Static HTML Export](https://nextjs.org/docs/advanced-features/static-html-export).
+
+> Currently here are limitations for usage of Static Export: 
+> 1. i18n not supported, so multilingual apps can't be exported.
+> 1. Visitor Identification not available.
+> 1. Request rewrites not supported.
+> 1. API Routes are not supported because they can't be prerendered to HTML.
+
+These instructions you should apply in order to run `next export`:
+1. Remove `i18n`:
+	1. Delete i18n configuration in `next.config.js`.
+	1. Define language in `package.json` using `config.language` which will be used during export.
+1. Remove `rewrites` in `next.config.js`.
+1. Define `getStaticPaths` in `[[...path]].tsx` and use:
+	```js
+	import { GetStaticPaths } from 'next';
+	import { graphQLSitemapService } from 'lib/graphql-sitemap-service';
+	import config from 'temp/config';
+	import packageConfig from '../../package.json';
```

Comment:
Could do `
import { config as packageConfig } from '../../package.json';` here, then use `packageConfig.language` below.

Reply:
@ambrauer Like it :)